.github/workflows/dockerpush.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and push Docker images
+
+on:
+  push:
+    paths:
+      # Match any VERSION file under /src folder
+      - 'src/**/VERSION'
+env:
+  DOCKER_REGISTRY_URL: iad.ocir.io
+  REPOSITORY: oracle/mushop
```

Comment:
This may be more convenient if be a secret, so we can change the variable without change the workflow file.